app/src/main/java/com/dsu/pocapplication/news/ui/login/repository/LoginRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.pocapplication.news.ui.login.repository
```

Comment:
repository package should not be a part of ui

Reply:
Please read and understand about repository pattern initially: https://developer.android.com/jetpack/docs/guide